plugins/modules/packaging/os/zypper.py

```diff
@@ -336,7 +338,9 @@ def get_cmd(m, subcommand):
     "puts together the basic zypper command arguments with those passed to the module"
     is_install = subcommand in ['install', 'update', 'patch', 'dist-upgrade']
     is_refresh = subcommand == 'refresh'
-    cmd = [m.get_bin_path('zypper', required=True), '--quiet', '--non-interactive', '--xmlout']
+    cmd = ['/usr/bin/zypper', '--quiet', '--non-interactive', '--xmlout']
```

Comment:
note: I don't know even know what zypper is, so I'm a bit unfamiliar just trying to help review.

Is it ok to hardcode the path here? It seems like previously there was a call to find the bin path; wondering what that did or why it'd be ok to remove it.

Reply:
This definitely has to be changed back.

Reply:
Yeah, that's my fault. I used the file I had installed locally as base and the repo's file contained different code.

Reply:
Fixed, also for the transactional-update executable